src/modules/function/declaration.rs

```diff
@@ -271,13 +277,74 @@ impl TranslateModule for FunctionDeclaration {
 impl DocumentationModule for FunctionDeclaration {
     fn document(&self, meta: &ParserMetadata) -> String {
         let mut result = vec![];
-        result.push(format!("## `{}`", self.name));
+        result.push(format!("## `{}`\n", self.name));
+
+        let references = self.create_reference(meta, &mut result);
+
         result.push("```ab".to_string());
         result.push(self.doc_signature.to_owned().unwrap());
         result.push("```\n".to_string());
         if let Some(comment) = &self.comment {
             result.push(comment.document(meta));
         }
+
+        if let Some(references) = references {
+            for reference in references {
+                result.push(reference);
+            }
+            result.push("\n".to_string());
+        }
+
         result.join("\n")
     }
 }
+
+impl FunctionDeclaration {
+    #[cfg(debug_assertions)]
+    fn create_reference(&self, meta: &ParserMetadata, result: &mut Vec<String>) -> Option<Vec<String>> {
+        let mut references = Vec::new();
+        let exe_path = env::current_exe()
+            .expect("Executable path not found");
+        let root_path = exe_path.parent()
```

Comment:
All path operations are now done via `Path` objects, so it's more reliable, and cross-platform.